app/templates/events/view/tickets/access-codes/create.hbs

```diff
@@ -0,0 +1,83 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="ui header column">
+      {{t 'Create an access code'}}
+    </div>
+  </div>
+  <div class="row">
+    <div class="sixteen wide column">
+      <form class="ui form">
+        <div class="field">
+          <label class="required">{{t 'Access Code'}}</label>
+          {{input type='number' name='access_code'}}
+        </div>
+        <div class="field">
+          <label class="required">{{t 'Number of Access Tickets'}}</label>
+          {{input type='number' name='number_of_access_tickets'}}
+        </div>
+        <div class="field">
+          <label class="required">{{t 'Status'}}</label>
+          <div>
+            {{input type='radio' name='status'}}
```

Comment:
This is **not** the right markup for radio buttons in semantic ui. It's been many weeks since this project started. And such simple things are still not being done correctly.